store/ivy.xml

```diff
@@ -138,5 +138,7 @@
   <dependency org="org.tukaani" name="xz" rev="1.9"/>
   <dependency org="com.drewnoakes" name="metadata-extractor" rev="2.16.0"/>
   <dependency org="com.adobe.xmp" name="xmpcore" rev="6.1.11"/>
+  <dependency org="org.apache.xmlgraphics" name="batik-i18n" rev="1.14"/>
```

Comment:
for these libs should we update the version here too https://github.com/Zimbra/zm-mailbox/blob/9bf90b194bebc6140918895f3ada586e8347dd04/store/build.xml#L292 ?